wesnoth-tsg.po

```diff
@@ -4597,6 +4577,12 @@ msgid ""
 "sorceries, I feel nothing but pity for the shambling shell you have become. "
 "Perhaps I can take solace in putting you out of your misery."
 msgstr ""
+"Skutečně žalostná je situace, do které jsi se dostal. Ty, kdysi moudrý elfí "
```

Comment:
```suggestion
"Situace, do které ses dostal, je skutečně žalostná. Ty, kdysi moudrý elfí "
```

Reply:
Ses je právně, ale to žalostná by mělo být na začátku, protože Deoran tím reaguje na předchozí narážku.
Ještě to předělám.